app/src/main/AndroidManifest.xml

```diff
@@ -18,26 +39,25 @@
             android:name=".meetups.activities.MeetUpCreation"
             android:exported="false"
             android:parentActivityName=".MainActivity"/>
-        <activity
-            android:name=".GreetingActivity"
-            android:exported="false" />
+     
 
         <activity
-            android:name=".ui.login.LoginActivity"
```

Comment:
Here I think you are changing the main activity when launching the app to "MainActivity" but it should remain at the "LoginActivity"

Reply:
the login activity don't let us access the rest of the application but we can't connect, so I bypassed it

Reply:
That is no problem when you work on your tasks, but before merging be sure to revert it so we avoid any branching to revert afterwards. You can also use the palfinder email to login with the google authentification

Reply:
i reverted it, but palfinder email is not in the database

Reply:
Yes this is because nobody loged in with this email. You could also use your own google account to log in. Thank you for reverting it :)